provisioning/local/gcp_kms.tf

```diff
@@ -0,0 +1,23 @@
+resource "google_kms_key_ring" "object_encryptor_keyring" {
+  name     = "${var.name_prefix}-object-encryptor"
+  location = "global"
+}
+
+resource "google_kms_crypto_key" "object_encryptor_key" {
+  name     = "${var.name_prefix}-object-encryptor"
+  key_ring = google_kms_key_ring.object_encryptor_keyring.id
+  purpose  = "ENCRYPT_DECRYPT"
+
+  lifecycle {
+    prevent_destroy = false
+  }
+
+  labels = {
+    "stack" = "${var.name_prefix}-object-encryptor"
+    "role"  = "object-encryptor"
+  }
```

Comment:
Koukam do dokumentace a `google` provider asi neumi globalni `labels` nakonfigurovany u `provider`, jako `aws`, co? Tzn budeme to muset davat ke kazdymu jednotlivimu resource? Jsou na to treba nejaky guidelines od SRE?

Reply:
jj, neumí, guidlines nevím

Reply:
je to pryč https://keboolaglobal.slack.com/archives/C05H32UUJK1/p1693219249360189?thread_ts=1693218546.489069&cid=C05H32UUJK1